webapps/frontend/public/app/admin/locales/en.json

```diff
@@ -172,6 +172,7 @@
     "EXECUTION_METRICS_ANNUAL_FORMAT": "{{from}} to {{to}}",
     "EXECUTION_METRICS_ANNUAL_FORMAT_CURRENT": "{{from}} up to today",
     "EXECUTION_METRICS_ANNUAL_TABLE_EMPTY": "Annual usage not available yet.",
+    "EXECUTION_METRICS_FETCH_DATA_ERROR_MESSAGE": "Execution metrics data fetching failed: {{msg}}",
```

Comment:
🙃 I think it would be easier to read the error message with a different word order: 
"Fetching execution metrics data failed: {{msg}}"
or 
"Failed to fetch execution metrics data: {{msg}}"